### PR TITLE
Chore/ppdse 2462 release fix with local storage

### PR DESCRIPTION
**Pull request by baburay23**
PPDSC-2462

**What**

1. Background - adding local storage data to the cypress in order to make tests pass. In the past the tests passed with just a set cookie but changes have been made in sourcepoint. We need to obtain data in local storage as well.This way the consent modal does not appear as we have set the cookie and local storage. So it does not pop up and block the tests which previously caused them to fail
2. What did you do
3. What does the reviewers should expect

**I have done:**
- [ ] Written unit tests against changes
- [ ] Written functional tests against the component and/or NewsKit site
- [ ] Updated relevant documentation

**I have tested manually:**
- [ ] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [ ] The screen reader reads and flows through the elements as expected.
- [ ] There are no new errors in the browser console coming from this PR.
- [ ] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
- [ ] The Playground feature is working as expected

**Before:**

**After:**

**Who should review this PR:**

**How to test:**

### CODE REVIEW

**Comment by newskit-engineering** [comment]
You can preview these changes on:
  Documentation Site: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdse-2462-release-fix-with-local-storage
  Storybook: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdse-2462-release-fix-with-local-storage/storybook

File: cypress/fixtures/consent/local-storage/_sp_user_consent_5623.json [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+          "name": "Branch Metrics, Inc.",
+          "vendorType": "CUSTOM"
+        },
+        {
+          "_id": "5f745ab96f3aae0163740409",
+          "name": "Ipsos MORI",
+          "vendorType": "CUSTOM"
+        }
+      ],
+      "consentedPurposes": [
```
Comment (mstuartf) [inline cypress/fixtures/consent/local-storage/_sp_user_consent_5623.json:55]: Are the `grants`, `customVendorsResponse` and `consentedPurposes` definitely necessary or can they just be empty objects / lists?

File: cypress/fixtures/consent/local-storage/_sp_local_state.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (mstuartf) [inline cypress/fixtures/consent/local-storage/_sp_local_state.json:1]: There is a `consent-request.json` file in this dir. Is that used anywhere or can we delete it?

File: cypress/fixtures/consent/local-storage/_sp_user_consent_5623.json (hunk 1 above)
Comment (baburay23) [reply cypress/fixtures/consent/local-storage/_sp_user_consent_5623.json]: seems to work without them!

**Review by mstuartf — APPROVED** [review]

**Review by mutebg — APPROVED** [review]

**Pull request merged**